src/components/autocomplete/Autocomplete.vue

```diff
@@ -32,7 +32,8 @@
         <transition name="fade">
             <div
                 class="dropdown-menu"
-                :class="{ 'is-opened-top': !isListInViewportVertically }"
+                :class="{ 'is-opened-top': dropdownPosition==='top' ||
+                (dropdownPosition==='auto' && !isListInViewportVertically ) }"
```

Comment:
Would be better to use a computed property here.

Reply:
sure, will update it